modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -55,6 +57,16 @@ def pdf_structure(pdf_version)
       ).new(self)
     end
 
+    def assign_metadata
+      return unless api_version&.downcase == 'v2'
```

Comment:
We'll need to remember to shore up our metadata handling on `v0` controllers. I think getting this in now for SCs is worth it for gathering the data sooner than later, but we'll have to come up with a version-agnostic way of assigning metadata to the same keys based on differing schemas.

Reply:
Yeah, I had that thought, too. I started to work on it for a short amount of time, but it quickly became more complex, hence the limiting to V2 for now.

Reply:
I've filed [API-25814](https://vajira.max.gov/browse/API-25814) to update our api_version values - initially we were actually going to call the segmented APIs "v2", but we've moved away from that decision since then, so we should fix this